scripts/package.sh

```diff
@@ -27,22 +27,50 @@ if [ ! -f "${tmp_license_file}" ]; then
   echo "Cannot find a LICENSE file called ${filename} in the exporter repo";
   exit 1;
 fi
-cp "${tmp_license_file}" "${exporters_doc_dir}/${integration}-LICENSE"
+cp "${tmp_license_file}" "${integration_target}/${integration}-LICENSE"
 rm -rf "${tmp_dir}"
 
 echo "Packaging"
+mkdir -pv "${integration_target}/packages"
 if [ "$goos" == "windows" ]; then
-  powershell.exe -file "${root_dir}/scripts/win_msi_build.ps1" -arch amd64 -exporterName ${NAME} -version ${VERSION} -exporterGUID ${EXPORTER_GUID} -upgradeGUID ${UPGRADE_GUID} -licenseGUID ${LICENSE_GUID}
+    powershell.exe -file "${root_dir}/scripts/win_msi_build.ps1" -arch amd64 -exporterName ${NAME} -version ${VERSION} -exporterGUID ${EXPORTER_GUID} -upgradeGUID ${UPGRADE_GUID} -licenseGUID ${LICENSE_GUID}
+
+    mkdir -p "${root_dir}/dist/New Relic/newrelic-infra/integrations.d"
+    cp "${integration_dir}/${NAME}-config.yml.sample" "${root_dir}/dist/New Relic/newrelic-infra/integrations.d/"
+
+    mkdir -p "${root_dir}/dist/New Relic/newrelic-infra/newrelic-integrations/bin"
+    cp "${integration_target}/bin/nri-${NAME}.exe" "${root_dir}/dist/New Relic/newrelic-infra/newrelic-integrations/bin"
+    cp "${integration_dir}/${NAME}-win-definition.yml" "${root_dir}/dist/New Relic/newrelic-infra/newrelic-integrations/" || true
+
+    mkdir -p "${root_dir}/dist/New Relic/newrelic-infra/logging.d"
+    cp "${integration_dir}/${NAME}-log-win.yml" "${root_dir}/dist/New Relic/newrelic-infra/logging.d/" || true
+
+    (
+      # Inside a subshell so we do not change $PWD to the rest of the script
+      cd "${root_dir}/dist"
+      7z a -tzip "${integration_dir}/target/packages/nri-${NAME}-amd64.${VERSION}.zip" "New Relic"
+    )
 else
-   cp "${goreleaser_file_template}" "${goreleaser_file}"
-   IFS=',' read -r -a goarchs <<< "$PACKAGE_LINUX_GOARCHS"
-   for goarch in "${goarchs[@]}"
-   do
-     echo  "Adding ${goarch} to goreleaser"
-     yq e -i ".builds[0].goarch += [ \"${goarch}\" ]" "${goreleaser_file}"
-   done
-
-   GORELEASER_CURRENT_TAG=${VERSION} ${goreleaser_bin} release --config "${goreleaser_file}" --rm-dist --snapshot
-   echo "Signing the packages"
-   bash ${root_dir}/scripts/sign.sh "${root_dir}" "${integration}"
+    cp "${goreleaser_file_template}" "${goreleaser_file}"
+    yq e -i ".nfpms[0].package_name = \"nri-${NAME}\"" "${goreleaser_file}"
```

Comment:
We might want to leave a comment here explaining why this is here. I left it in the goreleaser YAML template but it would be awesome to have it on both sides.

Reply:
Comment added here: ([cfa0b16](https://github.com/newrelic/newrelic-prometheus-exporters-packages/pull/147/commits/cfa0b164eb9ed6475557cae9c179fe005e28da60))